content/logs/index.md

```diff
@@ -18,72 +18,75 @@ Datadog's log management is currently in private beta. If you would like to appl
 init_config:
-
 instances:
     [{}]
 
-#The host of the Datadog intake server to send log data to
-log_dd_url: "intake.logs.datadoghq.com"
-#Port used to send the logs
-log_dd_port: 10514
-skip_ssl_validation: true
 #(mandatory)Same api_key than in datadog.conf
 api_key: <YOUR_DATADOG_API_KEY>
 #(optional): same hostname than in Datadog.conf - used for host tags and links with metrics
 #hostname:  
+#If you need to change the port used to send the log - default is secured
+#log_dd_port: 10516
+#If you do not want to use ssl to send your logs
+#skip_ssl_validation: false
 
 {{< /highlight >}}
 
 * [Restart your agent](https://help.datadoghq.com/hc/en-us/articles/203764515-Start-Stop-Restart-the-Datadog-Agent)
 
 ## Enabling log collection from integrations
 
-To start collecting logs for an integration, you need to uncomment the log sections in your integration yaml file, and then set its parameters values according to your infrastructure.
+To start collecting logs for an integration, you need to uncomment the log sections in your integration yaml file, and then set its parameters' values according to your infrastructure.
 
 If you don't find a log section in your file you need to use the custom file configuration below instead. 
 
 <div class="alert alert-warning">
-You might not find any log section in your yaml files during the private beta, however please find below the list of integration and their yaml file that are supported:
+You might not find any log section in your yaml files during the private beta, however please find below the list of integrations and their yaml files that are supported:
 </div>
 
 * Apache: [apache.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/apache/conf.yaml.example)
 * Haproxy: [haproxy.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/haproxy/conf.yaml.example)
 * IIS: [iis.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/iis/conf.yaml.example)
-* Java: [jmx.yaml.example](https://gist.github.com/NBParis/d977edbb2fc3fb4c1d47f8542661ceff)
+* Java: [java.yaml.example](https://gist.github.com/NBParis/d977edbb2fc3fb4c1d47f8542661ceff)
 * Mongo: [mongo.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/mongo/conf.yaml.example)
 * Nginx: [nginx.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/nginx/conf.yaml.example)
 
 ## Custom log collection
 
-The Datadog Agent can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, rename the `custom-logs.yaml.example` file to `custom-logs.yaml` and set these options:
+The Datadog Agent can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, rename the `custom-logs.yaml.example` file to `custom-logs.yaml` (or to some other unused `*.yaml` name) and set these options:
```

Comment:
"some other unused *.yaml name" seems off.  Lets pick specific files to include this in. Datadog Agent doesn't presently support arbitrary naming of config files so we should be consistent across agent types.

Reply:
👍